`make html` results in an error

**Describe the bug**
Running `make html` results in an error:
```
TypeError: __init__() got an unexpected keyword argument 'handle_io_errors'
```

**Steps to reproduce**
1. clone this repo
1. follow the instructions in the README:
    1. set up virtualenv inside the repo
    1. install libraries from requirements.txt and requirements-docs.txt
1. try to generate html docs
1. observe the failure

**Expected behavior**
No exception occurs when I try to generate the html docs.

**Debug logs**
This is my console log:
```console
william@william-ThinkPad-T450s: ~/Documents/wpyoga$ git clone https://github.com/boto/boto3.git
Cloning into 'boto3'...
remote: Enumerating objects: 20941, done.
remote: Counting objects: 100% (1808/1808), done.
remote: Compressing objects: 100% (616/616), done.
remote: Total 20941 (delta 1228), reused 1746 (delta 1192), pack-reused 19133
Receiving objects: 100% (20941/20941), 7.84 MiB | 2.83 MiB/s, done.
Resolving deltas: 100% (15359/15359), done.
william@william-ThinkPad-T450s: ~/Documents/wpyoga$ cd boto3
william@william-ThinkPad-T450s: ~/Documents/wpyoga/boto3$ virtualenv venv
created virtual environment CPython3.8.10.final.0-64 in 392ms
  creator CPython3Posix(dest=/home/william/Documents/wpyoga/boto3/venv, clear=False, no_vcs_ignore=False, global=False)
  seeder FromAppData(download=False, pip=bundle, setuptools=bundle, wheel=bundle, via=copy, app_data_dir=/home/william/.local/share/virtualenv)
    added seed packages: pip==21.1.3, setuptools==57.1.0, wheel==0.36.2
  activators BashActivator,CShellActivator,FishActivator,PowerShellActivator,PythonActivator,XonshActivator
william@william-ThinkPad-T450s: ~/Documents/wpyoga/boto3$ . venv/bin/activate
(venv) william@william-ThinkPad-T450s: ~/Documents/wpyoga/boto3$ pip3 install -r requirements.txt 
Obtaining botocore from git+git://github.com/boto/botocore.git@develop#egg=botocore (from -r requirements.txt (line 1))
  Cloning git://github.com/boto/botocore.git (to revision develop) to ./venv/src/botocore
  Running command git clone -q git://github.com/boto/botocore.git /home/william/Documents/wpyoga/boto3/venv/src/botocore
Obtaining jmespath from git+git://github.com/boto/jmespath.git@develop#egg=jmespath (from -r requirements.txt (line 2))
  Cloning git://github.com/boto/jmespath.git (to revision develop) to ./venv/src/jmespath
  Running command git clone -q git://github.com/boto/jmespath.git /home/william/Documents/wpyoga/boto3/venv/src/jmespath
  Running command git submodule update --init --recursive -q
Obtaining s3transfer from git+git://github.com/boto/s3transfer.git@develop#egg=s3transfer (from -r requirements.txt (line 3))
  Cloning git://github.com/boto/s3transfer.git (to revision develop) to ./venv/src/s3transfer
  Running command git clone -q git://github.com/boto/s3transfer.git /home/william/Documents/wpyoga/boto3/venv/src/s3transfer
Collecting nose==1.3.3
  Using cached nose-1.3.3-py3-none-any.whl
Collecting mock==1.3.0
  Using cached mock-1.3.0-py2.py3-none-any.whl (56 kB)
Collecting wheel==0.24.0
  Using cached wheel-0.24.0-py2.py3-none-any.whl (63 kB)
Collecting python-dateutil<3.0.0,>=2.1
  Using cached python_dateutil-2.8.2-py2.py3-none-any.whl (247 kB)
Collecting urllib3<1.27,>=1.25.4
  Using cached urllib3-1.26.6-py2.py3-none-any.whl (138 kB)
Collecting six>=1.7
  Using cached six-1.16.0-py2.py3-none-any.whl (11 kB)
Collecting pbr>=0.11
  Using cached pbr-5.6.0-py2.py3-none-any.whl (111 kB)
Installing collected packages: six, urllib3, python-dateutil, jmespath, pbr, botocore, wheel, s3transfer, nose, mock
  Running setup.py develop for jmespath
  Running setup.py develop for botocore
  Attempting uninstall: wheel
    Found existing installation: wheel 0.36.2
    Uninstalling wheel-0.36.2:
      Successfully uninstalled wheel-0.36.2
  Running setup.py develop for s3transfer
Successfully installed botocore-1.21.2 jmespath-0.10.0 mock-1.3.0 nose-1.3.3 pbr-5.6.0 python-dateutil-2.8.2 s3transfer-0.5.0 six-1.16.0 urllib3-1.26.6 wheel-0.24.0
(venv) william@william-ThinkPad-T450s: ~/Documents/wpyoga/boto3$ pip3 install -e .
Obtaining file:///home/william/Documents/wpyoga/boto3
Requirement already satisfied: botocore<1.22.0,>=1.21.2 in ./venv/src/botocore (from boto3==1.18.2) (1.21.2)
Requirement already satisfied: jmespath<1.0.0,>=0.7.1 in ./venv/src/jmespath (from boto3==1.18.2) (0.10.0)
Requirement already satisfied: s3transfer<0.6.0,>=0.5.0 in ./venv/src/s3transfer (from boto3==1.18.2) (0.5.0)
Requirement already satisfied: python-dateutil<3.0.0,>=2.1 in ./venv/lib/python3.8/site-packages (from botocore<1.22.0,>=1.21.2->boto3==1.18.2) (2.8.2)
Requirement already satisfied: urllib3<1.27,>=1.25.4 in ./venv/lib/python3.8/site-packages (from botocore<1.22.0,>=1.21.2->boto3==1.18.2) (1.26.6)
Requirement already satisfied: six>=1.5 in ./venv/lib/python3.8/site-packages (from python-dateutil<3.0.0,>=2.1->botocore<1.22.0,>=1.21.2->boto3==1.18.2) (1.16.0)
Installing collected packages: boto3
  Running setup.py develop for boto3
Successfully installed boto3-1.18.2
(venv) william@william-ThinkPad-T450s: ~/Documents/wpyoga/boto3$ pip3 install -r requirements-docs.txt 
Obtaining botocore from git+git://github.com/boto/botocore.git@develop#egg=botocore (from -r requirements.txt (line 1))
  Updating ./venv/src/botocore clone (to revision develop)
  Running command git fetch -q --tags
  Running command git reset --hard -q c329fd23d3672b724a6dbd152f738de0b4d5a7ac
Obtaining jmespath from git+git://github.com/boto/jmespath.git@develop#egg=jmespath (from -r requirements.txt (line 2))
  Updating ./venv/src/jmespath clone (to revision develop)
  Running command git fetch -q --tags
  Running command git reset --hard -q 599b0f7ea650a60f7512233301e70ff8151de0f0
  Running command git submodule update --init --recursive -q
Obtaining s3transfer from git+git://github.com/boto/s3transfer.git@develop#egg=s3transfer (from -r requirements.txt (line 3))
  Updating ./venv/src/s3transfer clone (to revision develop)
  Running command git fetch -q --tags
  Running command git reset --hard -q 3bbb99fec6bf2ef3d71afffeb53311a8e6e57f08
Collecting markupsafe<2.0,>=1.1
  Using cached MarkupSafe-1.1.1-cp38-cp38-manylinux2010_x86_64.whl (32 kB)
Collecting jinja2<3.0,>=2.3
  Using cached Jinja2-2.11.3-py2.py3-none-any.whl (125 kB)
Collecting Sphinx<1.3,>=1.1.3
  Using cached Sphinx-1.2.3-py3-none-any.whl (1.2 MB)
Collecting guzzle_sphinx_theme<0.8,>=0.7.10
  Using cached guzzle_sphinx_theme-0.7.11-py3-none-any.whl
Requirement already satisfied: nose==1.3.3 in ./venv/lib/python3.8/site-packages (from -r requirements.txt (line 4)) (1.3.3)
Requirement already satisfied: mock==1.3.0 in ./venv/lib/python3.8/site-packages (from -r requirements.txt (line 5)) (1.3.0)
Requirement already satisfied: wheel==0.24.0 in ./venv/lib/python3.8/site-packages (from -r requirements.txt (line 6)) (0.24.0)
Requirement already satisfied: python-dateutil<3.0.0,>=2.1 in ./venv/lib/python3.8/site-packages (from botocore->-r requirements.txt (line 1)) (2.8.2)
Requirement already satisfied: urllib3<1.27,>=1.25.4 in ./venv/lib/python3.8/site-packages (from botocore->-r requirements.txt (line 1)) (1.26.6)
Requirement already satisfied: six>=1.7 in ./venv/lib/python3.8/site-packages (from mock==1.3.0->-r requirements.txt (line 5)) (1.16.0)
Requirement already satisfied: pbr>=0.11 in ./venv/lib/python3.8/site-packages (from mock==1.3.0->-r requirements.txt (line 5)) (5.6.0)
Collecting Pygments>=1.2
  Using cached Pygments-2.9.0-py3-none-any.whl (1.0 MB)
Collecting docutils>=0.10
  Using cached docutils-0.17.1-py2.py3-none-any.whl (575 kB)
Installing collected packages: markupsafe, Pygments, jmespath, jinja2, docutils, Sphinx, botocore, s3transfer, guzzle-sphinx-theme
  Attempting uninstall: jmespath
    Found existing installation: jmespath 0.10.0
    Uninstalling jmespath-0.10.0:
      Successfully uninstalled jmespath-0.10.0
  Running setup.py develop for jmespath
  Attempting uninstall: botocore
    Found existing installation: botocore 1.21.2
    Uninstalling botocore-1.21.2:
      Successfully uninstalled botocore-1.21.2
  Running setup.py develop for botocore
  Attempting uninstall: s3transfer
    Found existing installation: s3transfer 0.5.0
    Uninstalling s3transfer-0.5.0:
      Successfully uninstalled s3transfer-0.5.0
  Running setup.py develop for s3transfer
Successfully installed Pygments-2.9.0 Sphinx-1.2.3 botocore-1.21.2 docutils-0.17.1 guzzle-sphinx-theme-0.7.11 jinja2-2.11.3 jmespath-0.10.0 markupsafe-1.1.1 s3transfer-0.5.0
(venv) william@william-ThinkPad-T450s: ~/Documents/wpyoga/boto3$ cd docs
(venv) william@william-ThinkPad-T450s: ~/Documents/wpyoga/boto3/docs$ make html
sphinx-build -b html -d build/doctrees   source build/html
Making output directory...
Running Sphinx v1.2.3
/home/william/Documents/wpyoga/boto3/venv/lib/python3.8/site-packages/dateutil/parser/_parser.py:1207: UnknownTimezoneWarning: tzname PDT identified but not understood.  Pass `tzinfos` argument in order to correctly return a timezone-aware datetime.  In a future version, this will raise an exception.
  warnings.warn("tzname {tzname} identified but not understood.  "
loading pickled environment... failed: [Errno 2] No such file or directory: '/home/william/Documents/wpyoga/boto3/docs/build/doctrees/environment.pickle'
building [html]: targets for 347 source files that are out of date
updating environment: 347 added, 0 changed, 0 removed
reading sources... [  0%] guide/clients                                                                                                                                                                                                       
Exception occurred:
  File "/home/william/Documents/wpyoga/boto3/venv/lib/python3.8/site-packages/sphinx/environment.py", line 597, in __init__
    FileInput.__init__(self_, *args, **kwds)
TypeError: __init__() got an unexpected keyword argument 'handle_io_errors'
The full traceback has been saved in /tmp/sphinx-err-dau2ygne.log, if you want to report the issue to the developers.
Please also report this if it was a user error, so that a better error message can be provided next time.
A bug report can be filed in the tracker at <https://bitbucket.org/birkenfeld/sphinx/issues/>. Thanks!
make: *** [Makefile:45: html] Error 1
```

This is the log file mentioned above:
[sphinx-err-dau2ygne.log](https://github.com/boto/boto3/files/6846953/sphinx-err-dau2ygne.log)

It didn't work. The first error I got is when I tried to run `pip3 install -e .`:
```console
william@william-ThinkPad-T450s: ~/Documents/wpyoga$ mkdir boto3_topdir
william@william-ThinkPad-T450s: ~/Documents/wpyoga$ cd boto3_topdir/
william@william-ThinkPad-T450s: ~/Documents/wpyoga/boto3_topdir$ git clone https://github.com/boto/boto3.git
Cloning into 'boto3'...
remote: Enumerating objects: 21005, done.
remote: Counting objects: 100% (1872/1872), done.
remote: Compressing objects: 100% (658/658), done.
remote: Total 21005 (delta 1264), reused 1795 (delta 1214), pack-reused 19133
Receiving objects: 100% (21005/21005), 7.91 MiB | 6.50 MiB/s, done.
Resolving deltas: 100% (15395/15395), done.
william@william-ThinkPad-T450s: ~/Documents/wpyoga/boto3_topdir$ virtualenv venv
created virtual environment CPython3.8.10.final.0-64 in 565ms
  creator CPython3Posix(dest=/home/william/Documents/wpyoga/boto3_topdir/venv, clear=False, no_vcs_ignore=False, global=False)
  seeder FromAppData(download=False, pip=bundle, setuptools=bundle, wheel=bundle, via=copy, app_data_dir=/home/william/.local/share/virtualenv)
    added seed packages: pip==21.1.3, setuptools==57.1.0, wheel==0.36.2
  activators BashActivator,CShellActivator,FishActivator,PowerShellActivator,PythonActivator,XonshActivator
william@william-ThinkPad-T450s: ~/Documents/wpyoga/boto3_topdir$ . venv/bin/activate
(venv) william@william-ThinkPad-T450s: ~/Documents/wpyoga/boto3_topdir$ pip3 install -r boto3/requirements.txt 
Obtaining botocore from git+git://github.com/boto/botocore.git@develop#egg=botocore (from -r boto3/requirements.txt (line 1))
  Cloning git://github.com/boto/botocore.git (to revision develop) to ./venv/src/botocore
  Running command git clone -q git://github.com/boto/botocore.git /home/william/Documents/wpyoga/boto3_topdir/venv/src/botocore
Obtaining jmespath from git+git://github.com/boto/jmespath.git@develop#egg=jmespath (from -r boto3/requirements.txt (line 2))
  Cloning git://github.com/boto/jmespath.git (to revision develop) to ./venv/src/jmespath
  Running command git clone -q git://github.com/boto/jmespath.git /home/william/Documents/wpyoga/boto3_topdir/venv/src/jmespath
  Running command git submodule update --init --recursive -q
Obtaining s3transfer from git+git://github.com/boto/s3transfer.git@develop#egg=s3transfer (from -r boto3/requirements.txt (line 3))
  Cloning git://github.com/boto/s3transfer.git (to revision develop) to ./venv/src/s3transfer
  Running command git clone -q git://github.com/boto/s3transfer.git /home/william/Documents/wpyoga/boto3_topdir/venv/src/s3transfer
Collecting nose==1.3.3
  Using cached nose-1.3.3-py3-none-any.whl
Collecting mock==1.3.0
  Using cached mock-1.3.0-py2.py3-none-any.whl (56 kB)
Collecting wheel==0.24.0
  Using cached wheel-0.24.0-py2.py3-none-any.whl (63 kB)
Collecting python-dateutil<3.0.0,>=2.1
  Using cached python_dateutil-2.8.2-py2.py3-none-any.whl (247 kB)
Collecting urllib3<1.27,>=1.25.4
  Using cached urllib3-1.26.6-py2.py3-none-any.whl (138 kB)
Collecting six>=1.7
  Using cached six-1.16.0-py2.py3-none-any.whl (11 kB)
Collecting pbr>=0.11
  Using cached pbr-5.6.0-py2.py3-none-any.whl (111 kB)
Installing collected packages: six, urllib3, python-dateutil, jmespath, pbr, botocore, wheel, s3transfer, nose, mock
  Running setup.py develop for jmespath
  Running setup.py develop for botocore
  Attempting uninstall: wheel
    Found existing installation: wheel 0.36.2
    Uninstalling wheel-0.36.2:
      Successfully uninstalled wheel-0.36.2
  Running setup.py develop for s3transfer
Successfully installed botocore-1.21.5 jmespath-0.10.0 mock-1.3.0 nose-1.3.3 pbr-5.6.0 python-dateutil-2.8.2 s3transfer-0.5.0 six-1.16.0 urllib3-1.26.6 wheel-0.24.0
(venv) william@william-ThinkPad-T450s: ~/Documents/wpyoga/boto3_topdir$ pip3 install -e .
ERROR: File "setup.py" or "setup.cfg" not found. Directory cannot be installed in editable mode: /home/william/Documents/wpyoga/boto3_topdir
```

I tried to continue, but `make html` immediately exited with an error:
```console
(venv) william@william-ThinkPad-T450s: ~/Documents/wpyoga/boto3_topdir$ pip3 install -r boto3/requirements-docs.txt 
Obtaining botocore from git+git://github.com/boto/botocore.git@develop#egg=botocore (from -r boto3/requirements.txt (line 1))
  Updating ./venv/src/botocore clone (to revision develop)
  Running command git fetch -q --tags
  Running command git reset --hard -q 930cfaa975804562823c7d2338ec05e85a843027
Obtaining jmespath from git+git://github.com/boto/jmespath.git@develop#egg=jmespath (from -r boto3/requirements.txt (line 2))
  Updating ./venv/src/jmespath clone (to revision develop)
  Running command git fetch -q --tags
  Running command git reset --hard -q 599b0f7ea650a60f7512233301e70ff8151de0f0
  Running command git submodule update --init --recursive -q
Obtaining s3transfer from git+git://github.com/boto/s3transfer.git@develop#egg=s3transfer (from -r boto3/requirements.txt (line 3))
  Updating ./venv/src/s3transfer clone (to revision develop)
  Running command git fetch -q --tags
  Running command git reset --hard -q 8f02523b88c19f84d6f74905e21d209724eced69
Collecting markupsafe<2.0,>=1.1
  Using cached MarkupSafe-1.1.1-cp38-cp38-manylinux2010_x86_64.whl (32 kB)
Collecting jinja2<3.0,>=2.3
  Using cached Jinja2-2.11.3-py2.py3-none-any.whl (125 kB)
Collecting Sphinx<1.3,>=1.1.3
  Using cached Sphinx-1.2.3-py3-none-any.whl (1.2 MB)
Collecting guzzle_sphinx_theme<0.8,>=0.7.10
  Using cached guzzle_sphinx_theme-0.7.11-py3-none-any.whl
Requirement already satisfied: nose==1.3.3 in ./venv/lib/python3.8/site-packages (from -r boto3/requirements.txt (line 4)) (1.3.3)
Requirement already satisfied: mock==1.3.0 in ./venv/lib/python3.8/site-packages (from -r boto3/requirements.txt (line 5)) (1.3.0)
Requirement already satisfied: wheel==0.24.0 in ./venv/lib/python3.8/site-packages (from -r boto3/requirements.txt (line 6)) (0.24.0)
Requirement already satisfied: python-dateutil<3.0.0,>=2.1 in ./venv/lib/python3.8/site-packages (from botocore->-r boto3/requirements.txt (line 1)) (2.8.2)
Requirement already satisfied: urllib3<1.27,>=1.25.4 in ./venv/lib/python3.8/site-packages (from botocore->-r boto3/requirements.txt (line 1)) (1.26.6)
Requirement already satisfied: pbr>=0.11 in ./venv/lib/python3.8/site-packages (from mock==1.3.0->-r boto3/requirements.txt (line 5)) (5.6.0)
Requirement already satisfied: six>=1.7 in ./venv/lib/python3.8/site-packages (from mock==1.3.0->-r boto3/requirements.txt (line 5)) (1.16.0)
Collecting docutils>=0.10
  Using cached docutils-0.17.1-py2.py3-none-any.whl (575 kB)
Collecting Pygments>=1.2
  Using cached Pygments-2.9.0-py3-none-any.whl (1.0 MB)
Installing collected packages: markupsafe, Pygments, jmespath, jinja2, docutils, Sphinx, botocore, s3transfer, guzzle-sphinx-theme
  Attempting uninstall: jmespath
    Found existing installation: jmespath 0.10.0
    Uninstalling jmespath-0.10.0:
      Successfully uninstalled jmespath-0.10.0
  Running setup.py develop for jmespath
  Attempting uninstall: botocore
    Found existing installation: botocore 1.21.5
    Uninstalling botocore-1.21.5:
      Successfully uninstalled botocore-1.21.5
  Running setup.py develop for botocore
  Attempting uninstall: s3transfer
    Found existing installation: s3transfer 0.5.0
    Uninstalling s3transfer-0.5.0:
      Successfully uninstalled s3transfer-0.5.0
  Running setup.py develop for s3transfer
Successfully installed Pygments-2.9.0 Sphinx-1.2.3 botocore-1.21.5 docutils-0.17.1 guzzle-sphinx-theme-0.7.11 jinja2-2.11.3 jmespath-0.10.0 markupsafe-1.1.1 s3transfer-0.5.0
(venv) william@william-ThinkPad-T450s: ~/Documents/wpyoga/boto3_topdir$ cd boto3/docs
(venv) william@william-ThinkPad-T450s: ~/Documents/wpyoga/boto3_topdir/boto3/docs$ make html
sphinx-build -b html -d build/doctrees   source build/html
Making output directory...
Running Sphinx v1.2.3

Exception occurred:
  File "conf.py", line 17, in <module>
ModuleNotFoundError: No module named 'boto3'
The full traceback has been saved in /tmp/sphinx-err-thxe9e3g.log, if you want to report the issue to the developers.
Please also report this if it was a user error, so that a better error message can be provided next time.
A bug report can be filed in the tracker at <https://bitbucket.org/birkenfeld/sphinx/issues/>. Thanks!
make: *** [Makefile:45: html] Error 1
```

Btw I somehow think that we should be using a virtual environment inside of the boto3 repo dir itself, because it's expected to be there, looking at this part of `.gitignore`:
```
# Common virtualenv names
venv
env2
env3
```

Oh and btw this is my environment:
```
william@william-ThinkPad-T450s: ~/Documents/wpyoga/boto3_topdir/boto3$ python3 --version
Python 3.8.10
william@william-ThinkPad-T450s: ~/Documents/wpyoga/boto3_topdir/boto3$ pip3 --version
pip 21.1.3 from /home/william/.local/lib/python3.8/site-packages/pip (python 3.8)
```

--- a/docs/README.rst
+++ b/docs/README.rst
@@ -4,6 +4,8 @@ Documentation for boto3 can be found `here <https://boto3.amazonaws.com/v1/docum
 
 Generating Documentation
 ~~~~~~~~~~~~~~~~~~~~~~~~
+Note: `Botocore's <https://github.com/boto/botocore>`_ requirement-docs.txt must be installed prior to attempting the following steps.
+
 Sphinx is used for documentation. You can generate HTML locally with the
 following:
 

--- a/docs/README.rst
+++ b/docs/README.rst
@@ -4,7 +4,7 @@ Documentation for boto3 can be found `here <https://boto3.amazonaws.com/v1/docum
 
 Generating Documentation
 ~~~~~~~~~~~~~~~~~~~~~~~~
-Note: `Botocore's <https://github.com/boto/botocore>`_ requirement-docs.txt must be installed prior to attempting the following steps.
+Note: `Botocore's <https://github.com/boto/botocore/blob/develop/requirements-docs.txt>`_ requirement-docs.txt must be installed prior to attempting the following steps.
 
 Sphinx is used for documentation. You can generate HTML locally with the
 following:

--- a/docs/README.rst
+++ b/docs/README.rst
@@ -4,7 +4,7 @@ Documentation for boto3 can be found `here <https://boto3.amazonaws.com/v1/docum
 
 Generating Documentation
 ~~~~~~~~~~~~~~~~~~~~~~~~
-Note: `Botocore's <https://github.com/boto/botocore/blob/develop/requirements-docs.txt>`_ requirement-docs.txt must be installed prior to attempting the following steps.
+Note: Botocore's `requirement-docs.txt <https://github.com/boto/botocore/blob/develop/requirements-docs.txt>`_ must be installed prior to attempting the following steps.
 
 Sphinx is used for documentation. You can generate HTML locally with the
 following:

--- a/docs/source/guide/configuration.rst
+++ b/docs/source/guide/configuration.rst
@@ -83,6 +83,7 @@ In the following example, a proxy list is set up to use ``proxy.amazon.com``, po
 
     client = boto3.client('kinesis', config=my_config)
 
+Alternatively, you can use the ``HTTP_PROXY`` and ``HTTPS_PROXY`` environment variables to specify proxy servers and the ``NO_PROXY`` environment variable to override proxy servers set by ``HTTP_PROXY`` and  ``HTTPS_PROXY``.  Proxy servers specified using the ``Config`` object will override proxy servers specified using environment variables.
 
 .. _configure_proxies:
 

--- a/docs/source/guide/configuration.rst
+++ b/docs/source/guide/configuration.rst
@@ -83,7 +83,7 @@ In the following example, a proxy list is set up to use ``proxy.amazon.com``, po
 
     client = boto3.client('kinesis', config=my_config)
 
-Alternatively, you can use the ``HTTP_PROXY`` and ``HTTPS_PROXY`` environment variables to specify proxy servers and the ``NO_PROXY`` environment variable to override proxy servers set by ``HTTP_PROXY`` and  ``HTTPS_PROXY``.  Proxy servers specified using the ``Config`` object will override proxy servers specified using environment variables.
+Alternatively, you can use the ``HTTP_PROXY`` and ``HTTPS_PROXY`` environment variables to specify proxy servers.  The ``NO_PROXY`` environment variable can be used to override proxy servers set by ``HTTP_PROXY`` and  ``HTTPS_PROXY``.  Proxy servers specified using the ``proxies`` option in the ``Config`` object will override proxy servers specified using environment variables.
 
 .. _configure_proxies:
 

